formatting/availability_zones_test.go

```diff
@@ -0,0 +1,20 @@
+package formatting
+
+import "testing"
+
+var flagtests = []struct {
```

Comment:
Would be much nicer to have the test tables in the standard way of unit tests defined [here](https://github.com/giantswarm/fmt/blob/master/go/unit_tests.md) 👍

Reply:
Done